contributor-docs/release-guide.md

```diff
@@ -1356,6 +1346,41 @@ Also, update [the Wikipedia article on Apache Beam](https://en.wikipedia.org/wik
 
 At the end of the release, go to the GitHub milestones page and mark the recently released version as closed.
 
+#### Update the Java BOM
+
+Google releases a BOM that pins compatible versions of their Java libraries.
+After the release, try updating the BOM to the latest version.
+
+To do so, create a draft PR and run test suites following the instructions at
+https://github.com/apache/beam/blob/master/contributor-docs/java-dependency-upgrades.md.
+
+If there are no test failures, request review and merge the PR as normal.
```

Comment:
Given the state of our CI and tests, a little more active triage than just looking at the status.

Reply:
I had kinda covered it in the next sentence, but I updated the wording to be more explicit